component/core-jpa/src/main/java/org/exoplatform/social/core/jpa/storage/RDBMSIdentityStorageImpl.java

```
@@ -643,48 +658,94 @@ public int getIdentitiesByFirstCharacterOfNameCount(String providerId, ProfileFi
                                                                 final ProfileFilter profileFilter,
                                                                 SpaceMemberFilterListAccess.Type type,
                                                                 long offset, long limit) throws IdentityStorageException {
-
-    List<Long> relations = new ArrayList<>();
-    if (space != null) {
```

Is it intended to remove the nullity check on space ?

---

nope, we should throw an exception here instead of a null check, WDYT ?

---

done

---

:+1: